spec/lib/govuk_elements_form_builder/form_builder_spec.rb

```diff
@@ -48,6 +48,19 @@ def size(method, size)
       ]
     end
 
+    it 'supports attributes defined as a string' do
+      output = builder.send method, 'name', class: 'custom-class'
```

Comment:
OK, so I'd suggest removing the `, class: 'custom-class'` from this test, or having a second test for that behaviour.